Makefile

```diff
@@ -20,15 +20,10 @@
 FASTBUILD ?= n ## Set FASTBUILD=y (case-sensitive) to skip some slow tasks
 
 ## Image URL to use all building/pushing image targets
-STABLE_DOCKER_REPO ?= gcr.io/cluster-api-provider-aws
-MANAGER_IMAGE_NAME ?= cluster-api-aws-controller
-MANAGER_IMAGE_TAG ?= 0.0.3
-MANAGER_IMAGE ?= $(STABLE_DOCKER_REPO)/$(MANAGER_IMAGE_NAME):$(MANAGER_IMAGE_TAG)
-DEV_DOCKER_REPO ?= gcr.io/$(shell gcloud config get-value project)
-DEV_MANAGER_IMAGE ?= $(DEV_DOCKER_REPO)/$(MANAGER_IMAGE_NAME):$(MANAGER_IMAGE_TAG)
```

Comment:
Would it make sense to add a comment stating that these variables can be overridden?